src/scale.ts

```diff
@@ -611,7 +611,7 @@ export interface Scale<ES extends ExprRef | SignalRef = ExprRef | SignalRef> {
    *
```

Comment:
Optional: we could also advertise this capability somewhere in the docs or in the docstring? Currently it's not mentioned

https://vega.github.io/vega-lite/docs/scale.html#2-setting-the-range-property-to-an-array-of-valid-css-color-strings seems close but it targets `scale.range` rather than Scale.scheme